docs-source/spring/content/development/cli/_index.md

```diff
@@ -560,3 +560,179 @@ The following is an example development workflow using the CLI:
+       compile-logs: Compilation progress
+```
+
+   1. Use the `compile` command to upload and automatically start compilation using the following syntax:
+
+```cmd
+   oractl:>help compile
+       NAME
+       compile - Compile a service with GraalVM
+
+       SYNOPSIS
+              compile [--artifact-path String] --help 
+
+       OPTIONS
+              --artifact-path String
+              Service jar to compile location
+              [Mandatory]
+
+              --help or -h 
+              help for compile
+              [Optional]
+```
+
+   Since the compilation of a .jar file via the tool `native-image` does not support cross-compilation, so it must be on the same kind of platform where the application will run, this service guarantees a compilation in the same Operating System and CPU type where the service it will be executed on the Kubernetes cluster.
+
+   The Spring Boot application **pom.xml** with the plugin:
+
+```xml
+<plugin>
+       <groupId>org.graalvm.buildtools</groupId>
+       <artifactId>native-maven-plugin</artifactId>
+</plugin>
+```
+
+The project should be compiled on the developer desktop with GraalVM version 22.3 or later using an **mvn** command. For example:
+  
+```cmd
+  mvn -Pnative native:compile -Dmaven.test.skip=true
+```
+
+   This pre-compilation on your desktop needs to check if there are any issues on the libraries used in your Spring Boot microservice. In addition, your executable jar file must include AOT generated assets such as gnenerated classes and JSON hint files. For additional information, see [Converting Spring Boot Executable Jar] (https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html#native-image.advanced.converting-executable-jars).
+
+   The following is an example of the command output::
+
+```cmd
+     oractl:>compile --artifact-path /Users/cdebari/demo-0.0.1-SNAPSHOT.jar
+       uploading: /Users/cdebari/demo-0.0.1-SNAPSHOT.jar
+       filename: demo-0.0.1-SNAPSHOT.jar
+       return: demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b
+       return: Shell script execution started on: demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b
+       successfully start compilation of: demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b
+       oractl:>
+```
+
+   The following example shows the generated batch ID that must be used to retrieve the log files, download the compiled file, and purge the service instance:
+
+   **demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b**
+
+   If omitted, the last batch is considered by default.
+
+   2. Use the `compile-logs` command to get the logs that show the compilation progress:
+
+```cmd
+oractl:>help compile-logs
+NAME
+       compile-logs - Compilation progress
+
+SYNOPSIS
+       compile-logs --batch String --help 
+
+OPTIONS
+       --batch String
+       file ID returned from compile command. If not provided by default the last file
```

Comment:
Change to:

File ID returned from the compile command. If not provided by default, then the last file.